index.html

```diff
@@ -48,7 +50,200 @@ <h1 class="h1">I’m Patrick Ndimukulaga Glad to see you!</h1>
           </li>
         </ul>
       </div>
+      </div>
+    </section>
+    
+    <div class="simi">
+      <div class="title grid">
+       
+        <div>
+          <img src="./images/Snapshoot_Portfolio.png" alt="portfolio"/>
+        </div>
+        <div class="title-text">
+          <div class="t-text">
+            <div class="port">
+              <h4>Tonic</h4>
+
+              <ul class="ul">
+                <li class="first-child">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+          <div class="q-text">
+            A daily selection of privately personalized reads; no accounts
+            or sign-ups required.
+          </div>
```

Comment:
- [ ] Kindly change this div tag to a p tag, as this is part of the requirements, please make this adjustment for all the cards 👍🏿